CHANGES.md

```diff
@@ -14,6 +14,9 @@
 
 <!-- Changes that affect Black's preview style -->
 
+- Single-character closing docstring quotes are no longer moved to their own line as
+  this is invalid. This was a bug introduced in version 22.6.0. (#3165)
```

Comment:
```suggestion
  this is invalid. This was a bug introduced in version 22.6.0. (#3166)
```

Reply:
Oh lol, it's obvious that I prepared this PR a few hours ago. I forgot #3165 was opened :P